Tweak: Перевод способностей вампиров

Ohar opened the pull request:
## What Does This PR Do

Перевод всех способностей вампира

## Why It's Good For The Game

Улучшает погружение

## Changelog
:cl:
tweak: Перевод всех способностей вампира
/:cl:

Ohar committed:
еревод фраз торговых автоматов

Ohar committed:
Фразы торговых автоматов: мелкие правки

Ohar committed:
Merge branch 'ss220-space:master220' into master220

Ohar committed:
Merge branch 'ss220-space:master220' into master220

Ohar committed:
Перевод способностей вампира

Ohar committed:
Локализован шведский акцент

Ohar committed:
replacetextEx → replacetextEx_char
See https://github.com/ss220-space/Paradise/pull/647#pullrequestreview-878443663

Ohar committed:
swedish desc: change text

Ohar committed:
Round Statistics and Score translation

Ohar committed:
Merge pull request #1 from ss220-space/master220

Update from SS220

Ohar committed:
Merge branch 'master220' into vampire

# Conflicts:
#	code/game/gamemodes/vampire/vampire_powers.dm

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -139,13 +139,13 @@
 /obj/effect/proc_holder/spell/vampire/self/rejuvenate
-	name = "Rejuvenate"
-	desc= "Use reserve blood to enliven your body, removing any incapacitating effects."
+	name = "Омоложение"
```
Не надо сильно придумывать своё, на википедии "Восстановление"

Ohar replied:
Поправил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
 			if(!affects(target))
-				to_chat(user, "<span class='warning'>Your piercing gaze fails to knock out [target].</span>")
-				to_chat(target, "<span class='notice'>[user]'s feeble gaze is ineffective.</span>")
+				to_chat(user, "<span class='warning'>Ваш пронзительный взгляд не смог вырубить [target].</span>")
```
Лучше использовать не "вырубить", а "оглушить"/"ошеломить"

Ohar replied:
Исправил на «заворожить»

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
 			else
-				to_chat(user, "<span class='warning'>Your piercing gaze knocks out [target].</span>")
-				to_chat(target, "<span class='warning'>You suddenly feel very weak.</span>")
+				to_chat(user, "<span class='warning'>Ваш пронзающий взгляд вырубает [target].</span>")
```
Лучше использовать не "вырубить", а "оглушить"/"ошеломить"

Ohar replied:
Да дело в том что он не оглушает (не глухота) и не ошеломляет (не как станбатон), он именно усыпляет.
Заменил на «завораживает».

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
-	name = "Diseased Touch (50)"
-	desc = "Touches your victim with infected blood giving them Grave Fever, which will, left untreated, causes toxic building and frequent collapsing."
-	gain_desc = "You have gained the Diseased Touch ability which causes those you touch to become weak unless treated medically."
+	name = "Заражающее касание (50)"
```
Требует обновление википедии на твой вариант

Ohar replied:
Поправил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
 /obj/effect/proc_holder/spell/vampire/targetted/disease/cast(list/targets, mob/user = usr)
 	for(var/mob/living/carbon/target in targets)
-		to_chat(user, "<span class='warning'>You stealthily infect [target] with your diseased touch.</span>")
+		to_chat(user, "<span class='warning'>Вы незаметно заражаете [target] заражающим касанием.</span>")
```
Чтобы убрать тавтологию, лучше заменить на "Вы незаметно инфицируете"

Ohar replied:
Поправил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
 /obj/effect/proc_holder/spell/vampire/mob_aoe/glare
-	name = "Glare"
-	desc = "A scary glare that incapacitates people for a short while around you."
+	name = "Вспышка"
```
Хоть я и не фанат такого варианта перевода, но он лучше подходит, чем "Блик". Требуется обновить вики на твой вариант

Ohar replied:
Поправил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -226,14 +226,14 @@
-	name = "Shapeshift (50)"
-	desc = "Changes your name and appearance at the cost of 50 blood and has a cooldown of 3 minutes."
-	gain_desc = "You have gained the shapeshifting ability, at the cost of stored blood you can change your form permanently."
+	name = "Оборотень (50)"
```
Замени на "Превращение"

Ohar replied:
Заменил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -252,14 +252,14 @@
-	name = "Chiropteran Screech (30)"
-	desc = "An extremely loud shriek that stuns nearby humans and breaks windows as well."
-	gain_desc = "You have gained the Chiropteran Screech ability which stuns anything with ears in a large radius and shatters glass in the process."
+	name = "Визг рукокрылых (30)"
```
Научное название летучих мышей - Chiroptera. "Рукокрылы" звучит смешно и странно одновременно. Рекомендую изменить перевод тут и в других местах на "Ультразвуковой визг" из-за того, что летучие мыши издают ультразвук

Ohar replied:
Да, а [по-русски они именно рукокрылые](https://ru.wikipedia.org/wiki/%D0%A0%D1%83%D0%BA%D0%BE%D0%BA%D1%80%D1%8B%D0%BB%D1%8B%D0%B5), не согласен с тобой.
Мы же переводим, сохраняя стилистику, правильно?

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -316,19 +316,19 @@
 			break
 	if(!C)
-		log_runtime(EXCEPTION("something bad happened on enthralling a mob, attacker is [user] [user.key] \ref[user]"), user)
+		log_runtime(EXCEPTION("При порабощении моба случилось что-то плохое. Атакующий: [user] [user.key] \ref[user]"), user)
```
"моба" на "существа"

Ohar replied:
Именно «моба», это админские логи.

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -356,19 +356,19 @@
 	serve_objective.owner = user.mind
 	serve_objective.target = H.mind
-	serve_objective.explanation_text = "You have been Enthralled by [user.real_name]. Follow [user.p_their()] every command."
+	serve_objective.explanation_text = "Вы были порабощены [user.real_name]. Выполняйте все [user.p_their()] приказы."
```
Я правильно понимаю, что тут ([user.p_their()]) будет "him" "her" "they"? Если да, то это кринж

Ohar replied:
Кринж, согласен.
Но иначе пока что никак

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -356,19 +356,19 @@
-	name = "Cloak of Darkness"
-	desc = "Toggles whether you are currently cloaking yourself in darkness."
-	gain_desc = "You have gained the Cloak of Darkness ability which when toggled makes you near invisible in the shroud of darkness."
+	name = "Плащ тьмы"
```
На иконке способности не плащ, поэтому лучше подойдет и по смыслу и по иконке "Покров тьмы"

Ohar replied:
Исправил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -356,19 +356,19 @@
-	gain_desc = "You have gained the Cloak of Darkness ability which when toggled makes you near invisible in the shroud of darkness."
+	name = "Плащ тьмы"
+	desc = "Переключается, маскируя вас в темноте"
+	gain_desc = "Вы получили способность Плащ тьмы, которая, будучи включённой, делает вас практически невидимыми под покровом тьмы"
```
"...вас практически невидимым в темноте"

Ohar replied:
Исправил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -386,12 +386,12 @@
-	name = "Summon Bats (50)"
-	desc = "You summon a pair of space bats who attack nearby targets until they or their target is dead."
-	gain_desc = "You have gained the Summon Bats ability."
+	name = "Дети ночи (50)"
```
Не понял от куда взялось такое название, если уверен, что хочешь его оставить - то обнови вики

Ohar replied:
Половина способностей вампира — отсылки к [вампиру из D&D](http://dm-stuff.ru/creature-catalog/vampire), призыв летучих мышей в том числе.
Вики обновил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -417,9 +417,9 @@
-	desc = "You take on the form of mist for a short period of time."
-	gain_desc = "You have gained the Mist Form ability which allows you to take on the form of mist for a short period and pass over any obstacle in your path."
+	name = "Облик тумана (50)"
+	desc = "Вы на короткое время превращаетесь в облачко тумана"
```
Уменьшительно ласкательная форма тут неуместна

Ohar replied:
Убрал

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -532,18 +532,17 @@
-	name = "Raise Vampires"
-	desc = "Summons deadly vampires from bluespace."
+	name = "Поднятие вампиров"
+	desc = "Призовите смертоносных вампиров из синего космоса."
```
Никаких синих космосов. Блюспейс. Мы уже обсуждали эту тему и я не понимаю, почему ты игнорируешь остальных и настаиваешь только на своём варианте перевода.

Ohar replied:
Да я тупо забыл про то что тут тоже был блюспейс. Исправил

botPurple commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -559,7 +558,7 @@
 	new /obj/effect/temp_visual/cult/sparks(user.loc)
 	var/turf/T = get_turf(user)
-	to_chat(user, "<span class='warning'>You call out within bluespace, summoning more vampiric spirits to aid you!</span>")
+	to_chat(user, "<span class='warning'>Вы взываете по синему космосу, призывая больше вампирских духов себе на помощь!</span>")
```
Синий космос на блюспейс

Ohar replied:
Исправил, переформулировал

botPurple reviewed (changes requested):
В своих лучших традициях, тебе нужно поработать над радикальностью перевода.

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825253398

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825253687

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825253959

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825255016

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825255096

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825255151

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825254864

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825253636

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825254176

Ohar committed:
Правка по ревью
https://github.com/ss220-space/Paradise/pull/622#discussion_r825254806

Ohar commented:
> В своих лучших традициях, тебе нужно поработать над радикальностью перевода.

Все вроде поправил

Ohar committed:
Fix mistype

Ohar committed:
Merge branch 'master220' into Swedish-l10n

Ohar committed:
Merge branch 'master220' into Swedish-l10n

Ohar committed:
Мелкие правки шведского акцента

Ohar committed:
Merge remote-tracking branch 'origin/Swedish-l10n' into Swedish-l10n

Ohar committed:
УБраны большая часть «ь» и «й», они сильно забивают речь

Ohar committed:
Упрощение шведского акцента

Ohar committed:
Добавлена функция склонения числительных

Ohar committed:
scoreboard: исправлены склонения числительных

Ohar committed:
Склонение числительных: Теперь правильно обрабатываются числа больше сотни

Ohar committed:
Склонение числительных: Добавлено значение по-умолчанию

Ohar committed:
Update code/__HELPERS/pronouns.dm

Co-authored-by: Vallat <vox.crit2013@gmail.com>

Ohar committed:
Merge branch 'master220' into Round_statistics

Ohar committed:
Merge branch 'master220' into Swedish-l10n

Ohar committed:
Merge remote-tracking branch 'origin/Round_statistics' into Round_statistics

Ohar committed:
Функция склонения оптимизирована и упрощена

Ohar commented:
> В своих лучших традициях, тебе нужно поработать над радикальностью перевода.

@botPurple Вроде всё поправил

Ohar committed:
Merge remote-tracking branch 'ss220/master220' into vampire

Ohar committed:
Merge branch 'Swedish-l10n' into vampire

Ohar committed:
Merge branch 'Round_statistics' into vampire

Ohar committed:
Функция гендеризации «genderize_ru»

Ohar committed:
Вампир: перевод стадий сытости

Ohar committed:
Вампир: задачи набора крови

Ohar committed:
Вампир: перевод админских действий

Ohar committed:
Вампир: перевод сообщений чата и итогов раунда

Ohar commented:
Добавил ещё перевод сообщений в чате
@botPurple  Проведи ревью, пожалуйста

Ohar committed:
declension_ru: исправлена обработка чисел, оканчивающихся на «4»

Ohar committed:
Merge branch 'Swedish-l10n' into Round_statistics

Ohar committed:
Правка была не нужна, код был правильный  😃
![изображение](https://user-images.githubusercontent.com/3854741/159117453-2b677ecc-9b28-4768-b077-52486d03e04f.png)

Ohar committed:
declension_ru: обработка исключений, проверка на целое число

Ohar committed:
declension_ru: нецелые числа и не числа → двойственное число

Ohar committed:
Update code/__HELPERS/pronouns.dm

Co-authored-by: Vallat <vox.crit2013@gmail.com>

Ohar committed:
Окно итогов раунда: мелкие оптимизации и правки

Ohar committed:
Merge remote-tracking branch 'origin/Round_statistics' into Round_statistics

Ohar committed:
Добыто руды: теперь тоже склоняется

Ohar committed:
Update code/game/gamemodes/scoreboard.dm

Co-authored-by: Aziz Chynaliev <azizonkg@gmail.com>

Ohar committed:
Слова теперь не разбиты на части

Ohar committed:
Merge remote-tracking branch 'origin/Round_statistics' into Round_statistics

Ohar committed:
Merge branch 'master220' into vampire

Ohar committed:
Merge branch 'Round_statistics' into vampire

Ohar committed:
Merge branch 'master220' into vampire

Ohar committed:
Fix code style

TrashDoxx commented on code/game/gamemodes/vampire/vampire.dm:
```diff
@@ -180,21 +181,21 @@
-You are weak to holy things and starlight. Don't go into space and avoid the Chaplain, the chapel and especially Holy Water."}
+		dat = "<span class='danger'>Вы — вампир!</span><br>"
+	dat += {"Чтобы укусить кого-то, нацельтесь в голову, выберите намерение вреда (4) и ударьте пустой рукой. Пейте кровь, чтобы получать новые силы.
+Вы уязвимы перед святость и звёздным светом. Не выходите в космос, избегайте священника, церкви и, особенно, святой воды."}
```
```suggestion
Вы уязвимы перед святостью и звёздным светом. Не выходите в космос, избегайте священника, церкви и, особенно, святой воды."}
```

TrashDoxx commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -226,14 +226,14 @@
-	gain_desc = "You have gained the shapeshifting ability, at the cost of stored blood you can change your form permanently."
+	name = "Превращение (50)"
+	desc = "Изменяет ваше имя и внешность, тратя 50 крови, с откатом в 3 минуты."
+	gain_desc = "Вы получили способность навсегда обернуться другим обликом, тратя 50 крови."
```
Трижды пишем конкретные показатели, ну такое.
```suggestion
	gain_desc = "Вы получили способность навсегда обернуться другим обликом, затратив часть накопленной крови."
```

TrashDoxx commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -285,25 +285,25 @@
-	gain_desc = "You have gained the Enthrall ability which at a heavy blood cost allows you to enslave a human that is not loyal to any other for a random period of time."
+	name = "Порабощение (300)"
+	desc = "Вы используете большую часть своей силы, вынуждая тех, кто ещё никому не служит, служить только вам."
+	gain_desc = "Вы получили способность Порабощение, которая тратит много крови, но позволяет вам поработить человека, который ещё никому не служит, на случайный период времени."
```
Везде с малой, тут с большой.
```suggestion
	gain_desc = "Вы получили способность порабощения, которая тратит много крови, но позволяет вам поработить человека, который ещё никому не служит, на случайный период времени."
```

TrashDoxx commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -356,19 +356,19 @@
-	gain_desc = "You have gained the Cloak of Darkness ability which when toggled makes you near invisible in the shroud of darkness."
+	name = "Покров тьмы"
+	desc = "Переключается, маскируя вас в темноте"
+	gain_desc = "Вы получили способность Покров тьмы, которая, будучи включённой, делает вас практически невидимыми в темноте"
```
```suggestion
	gain_desc = "Вы получили способность к маскировке, которая, будучи включённой, делает вас практически невидимым в темноте."
```

TrashDoxx commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -469,9 +469,9 @@
-	gain_desc = "You have gained the ability to shadowstep, which makes you disappear into nearby shadows at the cost of blood."
+	name = "Шаг в тень (20)"
+	desc = "Растворитесь в тенях"
+	gain_desc = "Вы получили способность шагать в тень, позволяющую вам, тратя кровь, исчезать в ближайшей тени."
```
```suggestion
	gain_desc = "Вы получили возможность скрываться в тенях, позволяющую вам, затратив часть крови, оказаться в ближайшей доступной тени."
```

TrashDoxx reviewed (changes requested):
Доёбываюсь до мелочей, ряя.

Ohar committed:
Update code/game/gamemodes/vampire/vampire.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/game/gamemodes/vampire/vampire_powers.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/game/gamemodes/vampire/vampire_powers.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/game/gamemodes/vampire/vampire_powers.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/game/gamemodes/vampire/vampire_powers.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

TrashDoxx reviewed (approved):
Замечательная работа.

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
-	name = "Glare"
-	desc = "A scary glare that incapacitates people for a short while around you."
+	name = "Вспышка"
+	desc = "Жуткая вспышка, ненадолго ошеломляющая всех людей вокруг"
```
Раз уж назвали вспышкой вместо взгляда (про игру слов на английском, думаю, все поняли), надо хотя бы в описании указать, что эта вспышка из глаз, так как это важно. Я бы оставил вариант из вики "Вы сверкаете глазами, ненадолго ошеломляя всех людей вокруг".

Ohar replied:
Поправил

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
+	user.visible_message("<span class='warning'>Глаза [user] ослепительно вспыхивают!</span>")
 	if(istype(user:glasses, /obj/item/clothing/glasses/sunglasses/blindfold))
-		to_chat(user, "<span class='warning'>You're blindfolded!</span>")
+		to_chat(user, "<span class='warning'>Вас ослепляет!</span>")
```
У вас на глазах повязка!

Ohar replied:
И верно.
Поправил

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -226,14 +226,14 @@
 		target.adjustStaminaLoss(20)
-		to_chat(target, "<span class='warning'>You are blinded by [user]'s glare.</span>")
-		add_attack_logs(user, target, "(Vampire) Glared at")
+		to_chat(target, "<span class='warning'>Вы ослеплены вспышкой [user].</span>")
```
тут тоже стоит добавить "из глаз". Это поможет понять ингейм, что надев повязку на глаза вампиру
 вспышка вам будет не страшна.

Ohar replied:
Поправил

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -285,25 +285,25 @@
 	for(var/mob/living/target in targets)
-		user.visible_message("<span class='warning'>[user] bites [target]'s neck!</span>", "<span class='warning'>You bite [target]'s neck and begin the flow of power.</span>")
-		to_chat(target, "<span class='warning'>You feel the tendrils of evil invade your mind.</span>")
+		user.visible_message("<span class='warning'>[user] кусает [target] в шею!</span>", "<span class='warning'>Вы кусаете в шею [target]и начинаете выкачивать силу.</span>")
```
Не выкачивать силу, а наоборот. Это обратный процесс. "и начинаете передачу сил" - нейтральный вариант, работающий в обе стороны

Ohar replied:
Почему? Это текст для вампира, когда он сосёт кровь. Он буквально выкачивает силу из жертвы.

Ohar replied:
Поправил

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -356,19 +356,19 @@
-	gain_desc = "You have gained the Cloak of Darkness ability which when toggled makes you near invisible in the shroud of darkness."
+	name = "Покров тьмы"
+	desc = "Переключается, маскируя вас в темноте"
+	gain_desc = "Вы получили способность к маскировке, которая, будучи включённой, делает вас практически невидимым в темноте."
```
Может, стоит более высокопарно перевести? "во тьме" вместо "в темноте"

Ohar replied:
Я так и сделал, но [Бот, наоборот, попросил поменять](https://github.com/ss220-space/Paradise/pull/622#discussion_r825254864).

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
-	gain_desc = "You have gained the Diseased Touch ability which causes those you touch to become weak unless treated medically."
+	name = "Заражающее касание (50)"
+	desc = "Ваше касание инфицирует кровь жертвы, заражая её могильной лихорадкой. Пока лихорадку не вылечат, жертва будет с трудом держаться на ногах, а её кровь будет наполняться токсинами."
+	gain_desc = "Вы получили способность «Заражающее касание». Она позволит вам ослаблять тех, кого вы коснётесь до тех пор, пока их не вылечат."
```
Стоит привести все "Вы получили способность ... " к единому виду, а то у тебя где-то Название способности, где-то "Название способности", где-то название способности, где-то заменено название на схожие слова

Ohar replied:
Может и стоит но я делаю только перевод, а не улучшение.
Если начать улучшать, то тут можно надолго закопаться. Это надо делать потом, отдельно.

girafe969 replied:
Ну изначально на англе везде было "Вы получили способность Такую-то, которая..."
А в процессе перевода у тебя все пошло по пезде

Ohar replied:
Унифицировал чутка

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -170,54 +170,54 @@
 		target.help_shake_act(user)
 		if(!affects(target))
-			to_chat(user, "<span class='warning'>They seem to be unaffected.</span>")
+			to_chat(user, "<span class='warning'>Вам кажется, что могильная лихорадка не подействовала на [target].</span>")
```
откуда взялась могильная лихорадка? До этого это название нигде не всплывало

Ohar replied:
[Всплывало на вики](https://wiki.ss220.space/index.php/Руководство_по_вирусологии#Grave_Fever). 
Ну и в переводе болезней [она тоже будет так называться](https://github.com/ss220-space/Paradise/pull/685/files#diff-dd4ead50c8b0d7f00659c19432c680c8ab328148f7eb5a72f78c24da47cec5c4R2).

girafe969 replied:
Но будучи вампиром, ты вообще можешь не знать что такое могильная лихорадка и что твоя способность ей заражает, так как нигде в описании способности этого нет. Стоит либо добавить это в описание способности, либо убрать это отсюда

Ohar replied:
Понял о чём ты. Заменил на «заражающее касание».

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -559,7 +558,7 @@
 	new /obj/effect/temp_visual/cult/sparks(user.loc)
 	var/turf/T = get_turf(user)
-	to_chat(user, "<span class='warning'>You call out within bluespace, summoning more vampiric spirits to aid you!</span>")
+	to_chat(user, "<span class='warning'>Ваш зов расходится в блюспейсе, на помощь созывая других вампирских духов!</span>")
```
Мелкий доеб. "созывая на помощь" звучит лучше

Ohar replied:
Не совсем, потому что это нарушает ритм белого стиха.
Английский текст написан немного в отстранённой манере, в русском я это попытался передать белым стихом.

> Ваш зов расходится в блюспейсе, (−+−+−−−+−)
> на помощь созывая других вампирских духов! (−+−+−−−−+−+−+−)

Сравни

> Быстро он к сонму друзей отступил, избегающий смерти. (+−−+−−+−−+−−+−−+−)
> Словно как путник, увидев дракона в ущелиях горных, (+−−+−−+−−+−−+−−+−)

Хотя стихи у меня говно, да

girafe969 replied:
Ваш зов расходится в блуспейсе,
(-+-+---+-)
На помощь созывая других вампирских духов
(-+---+--+-+-+-)
В слове созывая ударение не на первом слоге)

girafe969 reviewed (changes requested):
Я тут накалякал. В некоторых местах больше пища для размышлений, чем конкретные правки.

Ohar committed:
Merge branch 'ss220-space:master220' into vampire

Ohar committed:
Вампиры: правки по ревью

Ohar committed:
Вампиры: правки перевода

Ohar committed:
Вампиры: правки перевода

girafe969 commented on code/game/gamemodes/vampire/vampire.dm:
```diff
@@ -78,30 +78,31 @@
+					text += " как [vampire.current.real_name]"
 			else
-				text += "body destroyed"
+				text += "[vampire.name] (тело было уничтожено"
```
[vampire.name] ( тут лишнее, оно уже есть в 85 строке

girafe969 commented on code/game/gamemodes/vampire/vampire.dm:
```diff
@@ -123,18 +124,18 @@
+					text += " как [Mind.current.real_name]"
 			else
-				text += "body destroyed"
+				text += "[Mind.name] (тело было уничтожено"
```
То же что с 94 строкой, уже есть в 129

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -139,13 +139,13 @@
 	..()
 	if(!gain_desc)
-		gain_desc = "You have gained \the [src] ability."
+		gain_desc = "Вы получили способность [src]."
```
«[src]».

girafe969 commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -13,13 +13,13 @@
 	..()
 	if(!gain_desc)
-		gain_desc = "You have gained \the [src] ability."
+		gain_desc = "Вы получили способность [src]."
```
«[src]». Чтобы уже всё в один вид привести

girafe969 reviewed (changes requested):
Красиво делаешь, мелкие правки.

Ohar committed:
Вампиры: правки по ревью

Ohar committed:
Вампиры: правки по ревью

Ohar committed:
Настройки: убран преждевременный перевод

Ohar committed:
Вампиры: добавлены пропущенные кавычки

girafe969 reviewed (approved):
Галка отдалась. Поздравим её и тебя

Ohar committed:
Вампиры: Правка бага

Ohar committed:
Вампиры: откат перевода админской Traitor Panel
https://discord.com/channels/617003227182792704/964182555815456848/964417388466540584

Ohar committed:
Merge branch 'ss220-space:master220' into vampire
